bin/main.ml

```diff
@@ -85,6 +84,20 @@ module Main = struct
       ()
 end
 
+module Log = struct
+  include Jbuilder.Log
+
+  let create common =
+    Log.create ~display:common.config.display ()
+end
+
+module Scheduler = struct
+  include Jbuilder.Scheduler
+
+  let go ?log ~common fiber =
+    Scheduler.go ?log ~config:common.config fiber
```

Comment:
I wonder if we should just initialize the log here from `~common`. I think that's what we always do anywhere.

Reply:
indeed, although I think there is a case where we call the scheduler twice